### PR TITLE
Added Prime Factors in Python 3

### LINKED ISSUE #545: Prime Factors in Python 3
### This issue is about Prime Factors algorithm implementation in Python 3

-   [x] Issue Title is in form `[Algo/DS Name] in [Language]` (ex:- BubbleSort in Java)

-   [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate

-   [x] Assign this issue to me (I want to work on this)

-   [x] I ensure that I am not already assigned to 2 or more issues
  
-   [x] I will send my PR only after I'm assigned to this issue by any maintainer

### PR DESCRIPTION
**Pull request by iamyajat**
**Fixes #545 [(Link)](https://github.com/FOSS-UCSC/FOSSALGO/issues/545)**

By submitting this pull request I confirm I've read and complied with the below declarations.

-   [x] I have read the [Contribution guidelines](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md) 
and I am confident that my PR reflects them.

-   [x] I have followed the [coding guidelines](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md#cs) 
for this project.

-   [x] My code follows the [skeleton code structure](https://github.com/FOSS-UCSC/FOSSALGO/blob/master/CONTRIBUTING.md#sample).

-   [x] This pull request has a descriptive title. 
For example, `Added {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.

-   [x] This pull request shall only be merged once the build passes.

-   [x] I have mentioned the issue number correctly (with hyperlink) in this pull request description.

### CODE REVIEW

File: algorithms/ar-prfac/python3/prime_factor.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+def primefactor(arr):
+    if arr<=0:
+        return
+
```
Comment (RavinduSachintha) [inline algorithms/ar-prfac/python3/prime_factor.py:4]: Codacy found an issue: [Trailing whitespace](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6335031)

**Comment by iamyajat** [comment]
Hey,
I have corrected the issue. Please check and let me know.
Thanks.

**Review by RavinduSachintha — APPROVED** [review]

**Comment by RavinduSachintha** [comment]
Nice work @iamyajat 
Can you add this contribution to the algorithm chart in the Main readme file? don't change anything else. and also don't make another PR. just push changes to here.

**Comment by iamyajat** [comment]
Hey @RavinduSachintha,
I have checked Prime Number for Python 3 in the table. Do I have to do something else?
Thanks.

**Review by RavinduSachintha — APPROVED** [review]

**Comment by RavinduSachintha** [comment]
@all-contributors please add @iamyajat for code and doc

**Comment by RavinduSachintha** [comment]
@iamyajat thank you for your contribution. keep it up :+1: 
and make sure to star this repository :wink:

**Pull request merged**